README.md

```diff
@@ -21,23 +23,30 @@ Consider this file name:
 What is the purpose of this file, and what is the purpose of the numbers at the beginning of its name?
 
 ```text
-Your answer...
+The file is a database migrate file that specifies the strcuture of an artists table using Ruby syntax. 
+It's called a migrate file because Ruby handles the differences between databases, so a Ruby app can 
+easily migrate from one "brand" of database to another.
+
+The numbers are a timestamp, indicating the file was "born" on 26 July 2015 at 2:50:27 PM. 
+I'm not really sure why Ruby thinks that fie in particular needs a time stamp, as opposed 
+to any other file.
 ```
 
 ### Question 3
 
 In a Rails application, how is the router related to controller actions?  
 
 ```text
-Your answer...
+The router sends a request to the appropriate controller, with the request coming 
+in the form of the URL requested. 
 ```
 
 ### Question 4
 
 Assuming our Tunr Rails app (1) has a Song model that belongs to an Artist model and (2) uses nested resources, which of the following helpers would create a URL that routes to `songs#new`? (Select one answer)  
 
 ```
-[] artist_song_path( @artist, @song ) 
+[x] artist_song_path( @artist, @song ) 
 [] new_artist_song_path( @artist )
 [] create_artist_song_path( @artist )
 [] new_artist_song_path( @artist, Song.all )
```

Comment:
The correct answer is `new_artist_song_path( @artist )`.

`artist_song_path( @artist, @song )` would take us to the show page for a given song.